docs/pages/upgrading/cloud-linux.mdx

```diff
@@ -164,16 +164,115 @@ automatic updates:
    $ sudo systemctl status teleport-upgrade
    ```
 
+## Version locking
```

Comment:
Currently, the default installation flow shown in Step 2 does not include version locking. I would make version locking the default path and:

- Illustrate version locking when we tell the user to install `teleport-ent` earlier in this guide (the Tabs in Step 2).
- Add a brief sentence next to the installation command snippet indicating that we recommend version locking to prevent unintended updates.

We can then add a `Details` box below the installation instructions with the content of the `Bypass version lock` section, since these are intended for a more niche subset of the target audience.

In general, I think it would make sense to integrate the version locking instructions into the rest of the guide, rather than organize it to appear as a separate alternative.

Reply:
The version lock is automatically enabled when the teleport-ent-updater v15.1.10+ is installed or updated. Action is not required by the user to enable the version lock.

For most users, this version lock should not change their workflow. But there can be some instances when a user may want to manually update the teleport-ent package, which would then require these extra flags/steps required.

Reply:
How about I put this under a FAQ section

Reply:
I think my comment may have came from a misconception about version locking. Is the behavior with version locking that:
- You specify a version of the Teleport package when installing via a package manager (as the original guide shows in the package manager instructions in Step 2)
- The updater locks the version specified and prevents manual updates

If that's the case, I think we can remove the "FAQ" heading and elevate "Version locking" and "Updating the updater" to H2s, then merge the PR. I don't think we need the "FAQ" heading since the headings below it make sense as H2s in their own right.

Reply:
Yup, that's the expected behavior. We're just trying to prevent any unintentional updates of the teleport agent after it has been installed. The updater will lock the version and handle all future updates.